undefined symbol: cusparseScsrgemm

Hi,

I'm trying to get the torch-geometry package to work and everything installs just fine.

But I'm getting following error during import from torch_sparse:
```
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/home/philipp/.local/share/virtualenvs/pytorch_geometric-4vUoAfdD/lib/python3.6/site-packages/torch_sparse/__init__.py", line 4, in <module>
    from .spspmm import spspmm
  File "/home/philipp/.local/share/virtualenvs/pytorch_geometric-4vUoAfdD/lib/python3.6/site-packages/torch_sparse/spspmm.py", line 7, in <module>
    import spspmm_cuda
ImportError: /home/philipp/.local/share/virtualenvs/pytorch_geometric-4vUoAfdD/lib/python3.6/site-packages/spspmm_cuda.cpython-36m-x86_64-linux-gnu.so: undefined symbol: cusparseScsrgemm
```
For me that can be reproduced with:
```bash
#!/bin/bash
pipenv --python 3.6
export PATH=/opt/cuda/bin:$PATH
export CPATH=/opt/cuda/include
pipenv install http://download.pytorch.org/whl/cu92/torch-0.4.1-cp36-cp36m-linux_x86_64.whl
pipenv install torchvision
pipenv install cffi
pipenv install torch-scatter
pipenv install torch-sparse
# pipenv install torch-cluster
# pipenv install torch-spline-conv
# pipenv install torch-geometric
pipenv run python -c "from torch_sparse import spmm"
```
More info:
```
$ pipenv run python -c "import torch; print(torch.__version__)"
0.4.1
```

I'm not sure if this is a problem on my side or not. Would appreciate your help.

Pulled from master and ran:
```
pipenv run python setup.py install
pipenv run python setup.py test
```

However I still get the same error, it's the same for all the tests:
```
ImportError while importing test module '/home/philipp/tmp/pytorch_sparse/test/test_coalesce.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
test/test_coalesce.py:2: in <module>
    from torch_sparse import coalesce
torch_sparse/__init__.py:5: in <module>
    from .spspmm import spspmm
torch_sparse/spspmm.py:7: in <module>
    import spspmm_cuda
E   ImportError: /home/philipp/tmp/pytorch_sparse/spspmm_cuda.cpython-36m-x86_64-linux-gnu.so: undefined symbol: cusparseScsrgemm
```

--- a/setup.py
+++ b/setup.py
@@ -13,13 +13,10 @@ cmdclass = {}
 
 if torch.cuda.is_available():
     ext_modules += [
-        CUDAExtension(
-            'spspmm_cuda',
-            ['cuda/spspmm.cpp', 'cuda/spspmm_kernel.cu'],
-            extra_link_args=['-lcusparse'],
-        ),
-        CUDAExtension('unique_cuda',
-                      ['cuda/unique.cpp', 'cuda/unique_kernel.cu'])
+        CUDAExtension('spspmm_cuda',
+                      ['cuda/spspmm.cpp', 'cuda/spspmm_kernel.cu'],
+                      extra_link_args=['-l', 'cusparse']
+                      )
     ]
     cmdclass['build_ext'] = BuildExtension
 

--- a/setup.py
+++ b/setup.py
@@ -15,7 +15,7 @@ if torch.cuda.is_available():
     ext_modules += [
         CUDAExtension('spspmm_cuda',
                       ['cuda/spspmm.cpp', 'cuda/spspmm_kernel.cu'],
-                      extra_link_args=['-l', 'cusparse']
+                      extra_link_args=['-lcusparse', '-l', 'cusparse'],
                       )
     ]
     cmdclass['build_ext'] = BuildExtension

